include/engine/hint.hpp

```diff
@@ -63,8 +63,8 @@ struct Hint
     friend std::ostream &operator<<(std::ostream &, const Hint &);
 };
 
-static_assert(sizeof(Hint) == 60 + 4, "Hint is bigger than expected");
-constexpr std::size_t ENCODED_HINT_SIZE = 88;
+static_assert(sizeof(Hint) == 56 + 4, "Hint is bigger than expected");
+constexpr std::size_t ENCODED_HINT_SIZE = 80;
```

Comment:
@lbud do you know why the hint sizes changed here? Was this part of a bad rebase?